### PR TITLE
Updated css - Solves #153

### PR DESCRIPTION
**Pull request by laurencereeds**

### CODE REVIEW

**Review by tillyninjaspace — CHANGES_REQUESTED** [review]
Can you fix the child key prop issues in red that show in the console?

File: src/components/Product.js [hunk 1]
```diff
@@ -2,17 +2,14 @@ import React from 'react';
...
-    const product = productList.find(singleProduct => Number(productId) === singleProduct.id);
+    // const {productList} = props
+    // const {productId} = useParams();
+    // const product = productList.find(singleProduct => Number(productId) === singleProduct.id);
 
 
     return (
         <div>
             <h1>Product</h1>
-            {
```
Comment (tillyninjaspace) [inline src/components/Product.js]: A bit worried about leaving this conditional out, will it cause a render issue? What are your thoughts, @zachgarrett24 ?

**Review by tillyninjaspace — COMMENTED** [review]
See my comments regarding taking out { product && 
for rendering. Does it show any errors on your side at any point?

**Review by tillyninjaspace — APPROVED** [review]
Thanks for clarifying on the { product && for the product.js file

**Review by zachgarrett24 — APPROVED** [review]
looks good

**Pull request merged**